TermsBanner/styles.scss

```diff
@@ -0,0 +1,25 @@
+@import '../css/settings';
+@import '../css/mixins/index';
+
+.banner {
+  background-color: map-get($colors, light-grey);
+  color: map-get($colors, grey-text);
+  line-height: 17px;
```

Comment:
Both the text and the box are a little outside the grid :)

<img width="431" alt="screen shot 2017-06-21 at 15 14 43" src="https://user-images.githubusercontent.com/381614/27386273-8cda1cfc-5695-11e7-9b40-5673292d0520.png">